src/cancellation.ts

```diff
@@ -1,3 +1,4 @@
+import axios, { CancelToken } from 'axios'
```

Comment:
This seems confusing. If we're importing the whole module anyway, can we just qualify both uses below? Or is there something subtle I'm missing about declaration syntax?

Reply:
This was a super quick and dirty merge of imports. I'll fix.

Reply:
Oh, I thought `axios` was importing the module name rather than the default. But that makes sense.

Reply:
Yep this is correct. `import * as axios` would import the whole module, `import axis, { CancelToken } from 'axios'` is equivalent to `import { default as axios, CancelToken } from 'axios'`.